### ISSUE TITLE
Redirect to new user profile URL after the user removes their slug settings

### ISSUE THREAD

**Issue by MrOrz** [cofacts/rumors-site#563]
From 20240110 test result https://g0v.hackmd.io/tiTBOmf_TaGYL9THZtQ9tg#%E6%9C%AA%E7%AB%9F%E9%A0%85%E7%9B%AE1

remove slug 後 user link 不會更新，按 F5 會壞掉 [name=cai]
  - 應該也要 redirect 到 ?id=xxx 的 page 避免此狀況
  - 但這比較少發生，可以另外開票做 [name=mrorz]

**Comment by belong112** [comment]
taking this issue~

**Comment by MrOrz** [comment]
During test, we found that "This user does not exist" still happens, but at least it's less likely to happen.
https://g0v.hackmd.io/@cofacts/meetings/%2FJBzxtPVYTbattxTcYTVRgA#%E6%9C%AA%E7%AB%9F%E9%A0%85%E7%9B%AE

**Comment by belong112** [comment]
Yeah, I think is the problem of waiting for mutation. This also happens before my change.
